### PR TITLE
Admin merchants

### LINKED ISSUE #86: User Story 47, Admin visits Merchant Index Page
```
As an admin user
When I visit the merchant's index page at "/merchants"
I see all merchants in the system
Next to each merchant's name I see their city and state
The merchant's name is a link to their Merchant Dashboard at routes such as "/admin/merchants/5"
I see a "disable" button next to any merchants who are not yet disabled
I see an "enable" button next to any merchants whose accounts are disabled
```

### Checklist:

- [ ] base code tests are written
- [ ] base code is written to pass tests
- [ ] edge case ("sad path") tests written (if required)
- [ ] edge case code written (if required)
- [ ] code is reviewed
- [ ] branch is merged
- [ ] all tests still pass

child of #89

depends on #88

### PR DESCRIPTION
**Pull request by nicovigil1**
closes #86 - adds index view for admin merchants page as well as the admin ability to enable or disable merchants

### CODE REVIEW

File: app/controllers/admin/merchants_controller.rb [hunk 1]
```diff
@@ -5,5 +5,21 @@ def index
...
+        user.switch_enabled
+
+        redirect_to admin_merchants_path
+    end 
+
+
+    private 
+
+    def toggle_params
+        params.permit(:enabled, :user_id)
```
Comment (tonyr729) [inline app/controllers/admin/merchants_controller.rb]: @nicovigil1, Strong params are used for user form inputs. 

This is admin functionality and is a button click which doesn't need strong params.

File: app/models/user.rb [hunk 2]
```diff
@@ -14,4 +14,11 @@ class User < ApplicationRecord
     def self.merchants
       where(role: 1)
     end
+
+    def switch_enabled
+      switch_boolean = !attributes["enabled"]
+      update(enabled: switch_boolean)
+      #assign_attributes({:enabled => !attributes["enabled"]})
```
Comment (tonyr729) [inline app/models/user.rb]: Remove this comment if not needed.

File: app/controllers/admin/merchants_controller.rb (hunk 1 above)
Comment (nicovigil1) [reply app/controllers/admin/merchants_controller.rb]: el fin!

File: app/models/user.rb (hunk 2 above)
Comment (nicovigil1) [reply app/models/user.rb]: nice catch!
